project/src/components/PrivateRoute/PrivateRoute.tsx

```diff
@@ -0,0 +1,19 @@
+import { Navigate } from 'react-router-dom';
+import { AppRoute, AuthorizationStatus } from '../../const';
+
+type PrivateRouteProps = {
+  authorizationStatus: AuthorizationStatus;
+  children: JSX.Element;
+}
+
+function PrivateRoute(props: PrivateRouteProps): JSX.Element {
```

Comment:
Лучше использовать деструктуризацию сразу, если пропсов много то в столбик
```suggestion
function PrivateRoute({ authorizationStatus, children }: PrivateRouteProps): JSX.Element {
```